### PR TITLE
docs(*) update network and firewall tutorial with stream proxying

### PR DESCRIPTION
**Pull request by bungle**
### Summary

Small update to network and firewall guide about `stream` proxying.

### CODE REVIEW

File: app/1.1.x/network.md [hunk 1]
```diff
@@ -11,23 +11,30 @@ In this section you will find a summary about the recommended network and firewa
...
+* `8000` for proxying HTTP traffic, and
+* `8443` for proxying HTTPS traffic
 
-These are the **only ports** that should be made available to your clients.
+See [proxy_listen] for more details on HTTP/HTTPS proxy listen options. For production environment it is common
+to change HTTP and HTTPS listen ports to `80` and `443`.
 
-### Management api
+Kong can also proxy TCP/TLS streams. The stream proxying is disabled by default. See [stream_listen] for
+additional details on stream proxy listen options, and how to enable it (if you plan to proxy other than
```
Comment (h3xar0n) [inline app/1.1.x/network.md]:
```suggestion
additional details on stream proxy listen options, and how to enable it (if you plan to proxy anything other than
```

File: app/1.1.x/network.md [hunk 2]
```diff
@@ -11,23 +11,30 @@ In this section you will find a summary about the recommended network and firewa
...
+HTTP/HTTPS traffic).
+
+In general the proxy ports are the **only ports** that should be made available to your clients.
+
+### Admin API
 
 This is the port where Kong exposes its management api. Hence in production this port should be firewalled to protect
```
Comment (h3xar0n) [inline app/1.1.x/network.md]:
```suggestion
This is the port where Kong exposes its management API. Hence in production this port should be firewalled to protect
```

File: app/1.1.x/network.md [hunk 3]
```diff
@@ -36,11 +43,28 @@ Below are the recommended firewall settings:
...
 * The upstream Services behind Kong will be available via the [proxy_listen] interface/port values.
   Configure these values according to the access level you wish to grant to the upstream Services.
-* If you are binding the Admin API to a public-facing interface (via [admin_listen]), then **protect** it to only allow trusted clients to access the Admin API.
-  See also [Securing the Admin API][secure_admin_api].
-* Your proxy will need have rules added for any TCP Stream listeners that you configure. For example, if you want Kong to manage traffic on port `4242`, your
-  firewall will need to be allow traffic on said port.
+* If you are binding the Admin API to a public-facing interface (via [admin_listen]), then **protect** it to only
+  allow trusted clients to access the Admin API. See also [Securing the Admin API][secure_admin_api].
+* Your proxy will need have rules added for any HTTP/HTTPS and TCP/TLS stream listeners that you configure.
+  For example, if you want Kong to manage traffic on port `4242`, your firewall will need to be allow traffic
```
Comment (h3xar0n) [inline app/1.1.x/network.md]:
```suggestion
  For example, if you want Kong to manage traffic on port `4242`, your firewall will need to allow traffic
```

File: app/1.1.x/network.md [hunk 4]
```diff
@@ -36,11 +43,28 @@ Below are the recommended firewall settings:
...
+  on said port.
+
+#### Transparent Proxying
+
+It is worth to mention about `transparent` listen option that you may apply to [proxy_listen]
```
Comment (h3xar0n) [inline app/1.1.x/network.md]:
```suggestion
It is worth mentioning that the `transparent` listen option may be applied to [proxy_listen]
```

File: app/1.1.x/network.md [hunk 5]
```diff
@@ -36,11 +43,28 @@ Below are the recommended firewall settings:
...
+and [stream_listen] configuration. With packet filtering such as `iptables` (Linux) or `pf` (macOS/BSDs)
+or with hardware routers/switches, you can specify pre-routing or redirection rules for TCP packets that
+allow you to mangle the original destination address and port. For example a HTTP request with a destination
+address of `10.0.0.1`, and a destination port of `80` can be redirected to `127.0.0.1` at port `8000`.
+To make this work, you need (with Linux) to add `transparent` listen option to Kong proxy,
```
Comment (h3xar0n) [inline app/1.1.x/network.md]:
```suggestion
To make this work, you need (with Linux) to add the `transparent` listen option to Kong proxy,
```

**Comment by kikito** [comment]
Note that this PR should now be made against `master`, not `release/1.1.x` (which was already merged into `master`)

**Pull request merged**